doc/crypto/BIO_read.pod

```diff
@@ -31,7 +42,10 @@ BIO_puts() attempts to write a NUL-terminated string B<buf> to BIO B<b>.
 
 =head1 RETURN VALUES
 
-All these functions return either the amount of data successfully read or
+BIO_read_ex() and BIO_write_ex() return 1 if data was successfully read or
+written, and 0 otherwise.
```

Comment:
Oh. Yes. I'm confusing myself with the SSL_read_ex() and SSL_write_ex() behaviour.